### PR TITLE
Update the "Install Netdata with kickstart.sh" doc

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This PR is related with netdata/documentation#22, it includes:

- [X] The two options for installing with `wget` and `curl` have been merged into a Tab element, a feature that Docusaurus provides
- [X] A note has been added to instruct the user to change the placeholders if he intends to claim to the Cloud
- [X] Ensured that links follow the new approach to use absolute GitHub links
- [X] Formatted the file and fixed a few minor typos, and changed markdown comments to either "tip" or "note" Docusaurus admonitions

What else could be changed:

- There is a section in the page named "Connect node to Netdata Cloud during installation". With the new Install box that is in https://github.com/netdata/learn/pull/1288, and the note added, we cover this option. Do we need this section?

### CODE REVIEW

File: packaging/installer/methods/kickstart.md [hunk 1]
```diff
@@ -7,39 +7,55 @@ learn_status: "Published"
...
+:::note
+If you plan to also Claim the node to Netdata Cloud,
+make sure to replace `YOUR_CLAIM_TOKEN` with the claim token of your space,
+and `YOUR_ROOM_ID` with the ID of the room you are willing to claim to.
+:::
 
 ## What does `kickstart.sh` do?
 
-The `kickstart.sh` script does the following after being downloaded and run using `sh`:
+The `kickstart.sh` script does the following after being downloaded and ran using `sh`:
```
Comment (cakrit) [inline packaging/installer/methods/kickstart.md:51]:
```suggestion
The `kickstart.sh` script does the following after being downloaded and run using `sh`:
```

Comment (cakrit) [inline packaging/installer/methods/kickstart.md:51]:
```suggestion
The `kickstart.sh` script does the following after being downloaded and run using `sh`:
```

**Review by cakrit — CHANGES_REQUESTED** [review]

File: packaging/installer/methods/kickstart.md [hunk 2]
```diff
@@ -94,22 +110,25 @@ should not need to use special values for any of these):
...
+You either need to run the command with root privileges or run it with the user that is running the agent.  
+More details: [Connect an agent without root privileges](https://github.com/netdata/netdata/blob/master/claim/README.md#connect-an-agent-without-root-privileges) section.
+:::
 
-To automatically claim nodes after installation: 
+To automatically claim nodes after installation:
 
 1. Sign in to [Netdata Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces)
-2. Go to the [Spaces management area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces)
+2. Go to the [Spaces management area](https://github.com/netdata/netdata/blob/master/docs/cloud/spaces.md#manage-spaces)
```
Comment (cakrit) [inline packaging/installer/methods/kickstart.md:123]: Don't send them there. Nodes tab and then add nodes is more convenient. 

image

For their first node, it's actually easier, the CTA is everywhere. Create a new space and you'll see.

File: packaging/installer/methods/kickstart.md [hunk 3]
```diff
@@ -94,22 +110,25 @@ should not need to use special values for any of these):
 
 ### Connect node to Netdata Cloud during installation
```
Comment (cakrit) [inline packaging/installer/methods/kickstart.md:111]: This whole section is extremely bad. Connecting during installation just means copy/paste the kickstart command. Doing it after installation is exactly the same, unless for some reason you want to use the claiming script directly. It doesn't make sense. the way it is.

**Review by cakrit — CHANGES_REQUESTED** [review]

File: packaging/installer/methods/kickstart.md (hunk 3 above)
Comment (Ancairon) [reply packaging/installer/methods/kickstart.md:111]: This is the section I noted in the PR's comments that could be removed, if we give them a note about it earlier in the doc, do we need to repeat it again?

File: packaging/installer/methods/kickstart.md (hunk 2 above)
Comment (Ancairon) [reply packaging/installer/methods/kickstart.md:123]: I haven't edited the info in this section, only links and formatting, let's figure out what we should do with this section in the comment below.

File: packaging/installer/methods/kickstart.md (hunk 3 above)
Comment (cakrit) [reply packaging/installer/methods/kickstart.md:111]: Remove it then. Re-read both the quick start and this one from the beginning and see if they make sense/are compatible alone and together.

**Comment by Ancairon** [comment]
@cakrit I read the section, and side by side with the quickstart, the only info that seemed valuable and wasn't repeated was the description of the Cloud options that get passed to the kickstart script.

I moved them to the "Optional parameters to alter your installation" section, and I didn't see any sorting of the items, so I placed them in the bottom of the list.

**Review by cakrit — APPROVED** [review]

**Pull request merged**